docs/get-started/apps-integrations.md

```diff
@@ -1,19 +1,19 @@
 ---
 id: apps-integrations
-title: Installing Apps and Integrations
-sidebar_label: Installing Apps
+title: Installing apps and Integrations
+sidebar_label: Installing apps
 description: Learn how to install apps to your Library and to multiple environments.
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('img/icons/cloud/apps.png')} alt="icon" width="50"/>
 
-Sumo Logic apps give you pre-configured searches and dashboards customized with your source configurations and populated in a folder selected by you. In this doc, you'll learn how to install apps from the App Catalog to your Library and how to manage them in multiple environments.
+Sumo Logic apps give you pre-configured searches and dashboards customized with your source configurations and populated in a folder selected by you. In this doc, you'll learn how to install apps from the app Catalog to your Library and how to manage them in multiple environments.
```

Comment:
App Catalog is the name of the UI element; casing must match


<img width="250" alt="Screenshot 2024-01-19 at 10 24 12 AM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/cdc0f1ba-cf61-4b51-a6d9-8ae1303ffdd6">



```suggestion
Sumo Logic apps give you pre-configured searches and dashboards customized with your source configurations and populated in a folder selected by you. In this doc, you'll learn how to install apps from the **App Catalog** to your Library and how to manage them in multiple environments.
```